src/app/directives/strongRoute/strong-route-active.directive.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  ChangeDetectorRef,
+  ContentChildren,
+  Directive,
+  ElementRef,
+  Input,
+  Optional,
+  QueryList,
+  Renderer2,
+} from "@angular/core";
+import {
+  IsActiveMatchOptions,
+  Router,
+  RouterLink,
+  RouterLinkActive,
+} from "@angular/router";
+import { StrongRouteDirective } from "./strong-route.directive";
+
+@Directive({
+  // eslint-disable-next-line @angular-eslint/directive-selector
+  selector: "a[strongRouteActive]",
```

Comment:
Are we creating another string route directive? Why?

Reply:
I could make a directive that combines the functionality of `StrongRouteActiveDirective` and `UrlActiveDirective` if you wanted, however I'm unsure what selector to use. I could also theoretically override the `RouterModule` with my own version of `RouterLinkActive`, however that seems dangerous and brittle